python/ttmlir/dialects/ttnn.py

```diff
@@ -3,5 +3,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from ._ttnn_ops_gen import *
-from ._ttnn_enum_gen import *
```

Comment:
I think we did something weird, I merged in #1194, which assumed this line was kept. I'll make a new PR to keep it in I guess.

Reply:
I think this was my bad. I cherry-pick your original code but not the new changes. Lemme know when you put up a new PR and ill approve.